src/components/n-file-select/n-file-select-view-model.ts

```diff
@@ -46,7 +47,10 @@ export class NFileSelectViewModel extends ComponentViewModel
     private get _mimeTypesList(): string { return this.getBound("mimeTypes"); }
     private get _maxFileSizeValue(): number | null { return TypeHelper.parseNumber(this.getBound("maxFileSize")); }
     private get _isMultiple(): boolean { return this.getBound("multiple") != null && this.getBound("multiple") === true; }
-
+    private get _hideLoadingOnProcessingFile(): boolean
```

Comment:
remove the whole use of dialog service all together. You are exposing the 2 events processing started and completed. Let the user of the component hook into those events and choose to show the dialog service or something else